### DISCUSSION TITLE
WTF is this sh*t with 'Recommended for you'

### DISCUSSION THREAD

**Discussion by PeterH220** [community/community#75184] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I don't want random anybody's on my home view, I don't care for your suggestions, who the fuck gave you idea to spam with that bullshit random nobodies. I don't care, don't be a twitter and other garbage, i don't want any suggestions.

**Comment by Xrenya** [comment, marked as answer]
@PeterH220 you can switch it off using the filter. 
![image](https://github.com/community/community/assets/51479797/8514e920-8225-4c09-95ae-b334559bb604)

**Comment by PeterH220** [comment]
Thank you.
